src/examples/index.jsx

```diff
@@ -2,8 +2,14 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
 import ScatterPlotExample from './scatterplot';
+import AxisExample from './axis';
 
 ReactDOM.render(
-  <ScatterPlotExample />,
+  <div>
+    <h1>Axis Examples</h1>
+    <AxisExample />
+    <h1>Scatter Plot Examples</h1>
```

Comment:
Nice! the examples are super bare bones and putting better semantics around our headers is a step in the right direction